src/software/ai/primitive/move_primitive.h

```diff
@@ -88,14 +100,14 @@ class MovePrimitive : public Primitive
      *
      * @return whether or not the dribbler should be enabled while moving
      */
-    bool isDribblerEnabled() const;
+    DribblerEnable isDribblerEnabled() const;
```

Comment:
Can we change this to `getDribblerStatus` or `getDribblerEnable`?

Reply:
If we go for the latter option, I'd prefer `getDribblerEnabled`

Reply:
Renamed

Reply:
@EvanMorcom I don't see your changes...

Reply:
@jonathanlew Pushed